### PR TITLE
story/VETHUB-25

### PR DESCRIPTION
**Pull request by ramonvega96**
To test:
1. yarn run dev
2. Navigate to http://localhost:5173/
3. Submit an answer to the question.

Verify:
1. Button is only clickable when an option is selected.
2. Once clicked, button style and content should change. Content should be "Next" and the color must be blue.
3. When "Next" button is clicked, question selection must be rebooted and you should be in the initial state.

Run tests:
1. yarn run test

### CODE REVIEW

File: tests/components/MCQButton.test.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    expect(wrapper.find(".mcq-button[disabled]").exists()).toBe(true);
+    expect(wrapper.find(".mcq-button[disabled]").classes()).toContain("submit");
+    expect(wrapper.find(".mcq-button[disabled]").text()).toBe("Submit");
+  });
+
+  it("Should enable MCQ button when option is selected", async () => {
+    const optionList = optionMount();
+    const option = optionList[1];
+    await option.trigger("click");
+    expect(wrapper.find(".mcq-button:not([disabled])").exists()).toBe(true);
```
Comment (pipw) [inline tests/components/MCQButton.test.ts]: Use `wrapper.get()`
"As a rule of thumb, always use get ..."
(see: https://test-utils.vuejs.org/api/#get)

Also other instances!

**Review by pipw — CHANGES_REQUESTED** [review]
Prefering `get()` over `find()` will make you code simpler in a lot of cases.

File: tests/components/MCQButton.test.ts (hunk 1 above)
Comment (ramonvega96) [reply tests/components/MCQButton.test.ts]: Fixed :)

**Review by pipw — APPROVED** [review]
LGTM!

**Pull request merged**